docs/pages/faq.js

```diff
@@ -86,7 +86,7 @@ padding 0 .. 12
             />
           </Box>
 
-          <Heading size="400">How can add a new Icon?</Heading>
+          <Heading size="400">How can add a new icon?</Heading>
           <Text>
             If you need an Icon that is not listed on our{' '}
```

Comment:
"icon" shouldn't be capitalized here either, since it's referring to an icon, not our Icon component

Reply:
Done!